acceptance/bundle/deployment/unbind/grants/script

```diff
@@ -0,0 +1,18 @@
+envsubst < databricks.yml.tmpl > databricks.yml
+
+cleanup() {
+    trace $CLI bundle destroy --auto-approve
+    if [[ -n "$schema_id" ]]; then
+        trace $CLI schemas delete $schema_id
+    fi
+    echo $?
+}
+trap cleanup EXIT
+
+trace $CLI bundle deploy
+schema_id=$($CLI bundle summary --output json | jq -r '.resources.schemas.schema_1.id')
+trace $CLI grants get schema $schema_id --output json | jq '.privilege_assignments[] | select(.principal == "account users")'
+
+trace $CLI bundle deployment unbind schema_1
+trace $CLI bundle deploy --var "suffix=another"
+trace $CLI grants get schema $schema_id --output json | jq '.privilege_assignments[] | select(.principal == "account users")'
```

Comment:
Please add a few comments describing the intent and what is supposed to happen.

It looks like the test will also pass if you don't unbind the grants on the original schema. Or would the deploy on L17 erase those, and return something different on L18?

Reply:
> Or would the deploy on L17 erase those, and return something different on L18?

Yes, this is what happens. I'll add the description of expectations